rmc/utils/mpc.py

```diff
@@ -1062,5 +1067,18 @@ def liftover_mpc(
             " to GRCh38."
         )
     ht = hl.read_table(mpc_release.versions[freeze].path)
+    # Deduplicate MPC annotation by keeping largest MPC value per variant (locus/alleles) combination
+    ht = dedup_annot(ht, annot="mpc", get_min=False)
+    # Rename transcript information
+    ht = ht.transmute(transcript_grch37=ht.transcript)
```

Comment:
Curious if `ht.rename` would be faster/slower or same?

Reply:
I'm not sure, I've never used rename. transmute seems a little more straightforward

Reply:
either should work, I was just curious :)